contracts/base/LoanTermsConsensus.sol

```diff
@@ -71,6 +69,12 @@ contract LoanTermsConsensus is Consensus, LoanTermsConsensusInterface {
         );
     }
 
+    /**
+        @notice Processes the loan response
+        @param ZeroCollateralCommon.LoanRequest request Struct of the protocol loan request
```

Comment:
It should be @param request struct of the protocol loan request

Reply:
fixed